src/Server.java

```diff
@@ -5,6 +5,21 @@
 class Server {
   public static void main(String[] args) {
     ServerSocket server = null;
+    
+    // Create Data structures. 
+    UserStorage users = new UserStorage();
+    Authenticator authenticator = new Authenticator(users);
+    RoomStorage rooms = new RoomStorage();
+    // TODO: create logger. 
+    // Logger logger = new Logger(rooms);
```

Comment:
`Logger` is implemented now https://github.com/NewtonGluten/CS401-Group-Four-Project/pull/4